dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,155 @@
+
+[[event-notifications-building-from-source]]
+== Building from Source
+
+. Clone the repository to your *_GOPATH_*:
++
+----
+$ git clone git@github.com:openshift/online.git $GOPATH/src/github.com/openshift/online
+----
+
+. Install the command:
++
+----
+$ cd GOPATH/src/github.com/openshift/online/notifications
+  make install
+----
+
+. Run it on your cluster:
++
+----
+$ GOPATH/bin/notifications -v 5 --provider smtp --smtpFrom noreply@example.com --smtpHost smtp.example.com
+----
++
+The command assumes the `admin.kubeconfig` is in one of the default expected
+locations. Otherwise, the `kubeconfig` path can be specified with an environment
+variable:
++
+----
+KUBECONFIG=/etc/openshift/master/admin.kubeconfig $GOPATH/bin/notifications -v 5 \
+--provider smtp --smtpFrom noreply@example.com --smtpHost smtp.example.com
+----
+
+[[deploying-on-a-cluster]]
+== Deploying on a Cluster
+
+As a *system:admin* user, use the provided template to create the necessary
+resources:
+
+----
+$ oc -n openshift-infra new-app -f ../templates/notifications.yaml
+----
+
+You can also build the image locally and deploy it to your cluster:
+
+----
+$ make release -e TARGET=dev
+    oc -n openshift-infra new-app -f ../templates/notifications.yaml -e IMAGE_NAME=openshift/notifications:latest
+----
+
+[[event-notifications-setting-preferences]]
+== Setting Preferences
+
+The notifications feature uses namespace annotations to determine whether to send
+and where to send notifications for resources within a namespace. Below is the
+specific annotation that is used:
+
+----
+openshift.io/notifications.emailAddress: "bob@example.com"
+----
+
+If the above annotation is not present, it is assumed that notifications are
+disabled for this namespace and no notifications are sent. The value of the
+annotation must be a valid email address.
+
+[[event-notifications-adding-new-dependencies]]
+== Adding New Dependencies
+
+Use this set of commands to set up an environment where dependencies can be
+added:
+
+----
+$ go get k8s.io/kubernetes
+$ cd $GOPATH/src/k8s.io/kubernetes
+$ git remote add github.com/openshift-remote https://github.com/openshift/kubernetes
+$ git fetch github.com/openshift-remote
+$ git clone git@github.com:openshift/ose.git $GOPATH/src/github.com/openshift/origin
+$ cd $GOPATH/src/github.com/openshift/origin
+$ git checkout -b accountant-$TAG $TAG  <1>
+$ hack/move-upstream.sh $LAST_BUMP...master  <2>
+$ hack/godep-restore.sh
+$ cd $GOPATH/src/github.com/openshift/online/accountant
+$ godep restore
+----
+<1> `$TAG` refers to the version of the product you want.
+<2> `$LAST_BUMP` is the commit ID of the last bump (kubernetes) commit in the
+product repository. You can use `git log --grep "bump"` after checking out the
+tag in the OpenShift Container Platform repository to find the bump ID. Use the
+commit ID, not the ID in the commit message.
+
+[[event-notifications-configuring-notifications]]
+== Configuring Notifications via the Web Console
+
```

Comment:
I think a screenshot of the UI and/or a description of how to navigate to it would be great by the section describing notification preferences.